frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx

```diff
@@ -150,14 +154,58 @@ const LetterBody: React.FC<HabitabilityLetterContentProps> = (props) => {
   return (
     <>
       <p>
-        <Trans id="laletterbuilder.habitability.intro">LETTER TEXT</Trans>
+        <Trans id="laletterbuilder.habitability.intro">
```

Comment:
So, an annoying thing with using lingui id tags is you have to change the id whenever you change the internal text to register that there is new english content that needs to be translated. I usually just add like a 1 at the end and then start incrementing. See how the messages.po file for english didn't register the new content?

Reply:
got it, thank you! and thanks for the heads up that we don't need it unless the text is >200 chars.

Reply:
Wait, I thought it did pick it up: 
<img width="732" alt="Screen Shot 2022-04-15 at 2 03 50 PM" src="https://user-images.githubusercontent.com/1595717/163605314-d0d09e34-0f4a-4ee9-b9ce-63df84dd1f98.png">

Reply:
see comment below—I think it didn't update it in the spanish file

Reply:
ahhhh in the spanish file. ok got it, fixing